libs/ui/src/admin/NewsCard/NewsCardBase.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC } from 'react'
+
+import { HStack, Image, Stack, Text, VStack } from '@chakra-ui/react'
+import { AiOutlineEye, AiOutlineLike, AiOutlineShareAlt } from 'react-icons/ai'
+import { BsBookmarkHeart } from 'react-icons/bs'
+
+import ActionButton from './ActionButton'
+import { NewsCardProps } from './types'
+
+export const NewsCardBase: FC<NewsCardProps> = ({
+  isFeatured,
+  isVertical,
```

Comment:
I would give variant `(horizontal | vertical)` instead of boolean value.

```suggestion
  variant = "vertical"
```

In the issue there is a responsiveness requirement:

- [ ] Horizontal variant should always look like vertical variant in small viewports.

```tsx
const isVertical = useBreakpointValue({
  base: true, // Always vertical in smaller viewports
  lg: variant === 'vertical' ? true : false,
})
```

Or the other way around

```tsx
const isHorizontal = useBreakpointValue({
  lg: variant === 'horizontal' ? true : false
})
```